rotkehlchen/externalapis/opensea.py

```diff
@@ -246,38 +226,35 @@ def _deserialize_nft(
             collection = None
             # NFT might not be part of a collection
             if 'collection' in entry:
-                saved_entry = self.collections.get(entry['collection']['name'])
+                saved_entry = self.collections.get(entry['collection'])
                 if saved_entry is None:
                     # we haven't got this collection in memory. Query opensea for info
                     self.gather_account_collections(account=owner_address)
                     # try to get the info again
-                    saved_entry = self.collections.get(entry['collection']['name'])
+                    saved_entry = self.collections.get(entry['collection'])
 
                 if saved_entry:
                     collection = saved_entry
                     if saved_entry.floor_price is not None:
                         floor_price = saved_entry.floor_price
                 else:  # should not happen. That means collections endpoint doesnt return anything
-                    collection_data = entry['collection']
-                    collection = Collection(
-                        name=collection_data['name'],
-                        banner_image=collection_data['banner_image_url'],
-                        description=collection_data['description'],
-                        large_image=collection_data['large_image_url'],
+                    raise UnknownAsset(
```

Comment:
Change this to `DeserializationError` instead since UnknownAsset is for when we don't have an asset locally

Reply:
And also add that to the docstring as by reading the docstring it seems as if the only thing this can raise is a RemoteError. But it does also potentially raise `DeserializationError` elsewhere too